utils/lists.ts

```diff
@@ -217,6 +217,11 @@ export const derivativesList: Record<string, string>[] = [
     description: "Loot DAO governance and gold token",
     url: "https://etherscan.io/address/0x32353a6c91143bfd6c7d363b546e62a9a2489a20#writeContract",
   },
+  {
+    name: "Adventure Gems (for loot)",
+    description: "Rare and powerful gems 💎 for Loot holders that give the owner one or more special powers",
```

Comment:
could you please make it short to fit in one line like others?

Reply:
Done, see latest commit

Reply:
It still breaks into two lines. Already deleted the fork though, so I'll create a new request to solve it